flocker/node/script.py

```diff
@@ -365,123 +358,6 @@ def get_configuration(options):
     return configuration
 
 
-def _zfs_storagepool(
-        reactor, pool=FLOCKER_POOL, mount_root=None, volume_config_path=None):
-    """
-    Create a ``VolumeService`` with a ``zfs.StoragePool``.
-
-    :param pool: The name of the ZFS storage pool to use.
-    :param bytes mount_root: The path to the directory where ZFS filesystems
-        will be mounted.
-    :param bytes volume_config_path: The path to the volume service's
-        configuration file.
-
-    :return: The ``VolumeService``, started.
-    """
-    if mount_root is None:
-        mount_root = FLOCKER_MOUNTPOINT
-    else:
-        mount_root = FilePath(mount_root)
-    if volume_config_path is None:
-        config_path = DEFAULT_CONFIG_PATH
-    else:
-        config_path = FilePath(volume_config_path)
-
-    pool = zfs.StoragePool(
-        reactor=reactor, name=pool, mount_root=mount_root,
-    )
-    api = VolumeService(
-        config_path=config_path,
-        pool=pool,
-        reactor=reactor,
-    )
-    api.startService()
-    return api
-
-
-class DeployerType(Names):
-    """
-    References to the different ``IDeployer`` implementations that are
-    available.
-
-    :ivar p2p: The "peer-to-peer" deployer - suitable for use with system like
-        ZFS where nodes interact directly with each other for data movement.
-    :ivar block: The Infrastructure-as-a-Service deployer - suitable for use
-        with system like EBS where volumes can be attached to nodes as block
-        devices and then detached (and then re-attached to other nodes).
-    """
-    p2p = NamedConstant()
-    block = NamedConstant()
-
-
-class BackendDescription(PClass):
```

Comment:
:question: No action required. Just documenting my review process:

Confirmed that no one was incorrectly import BackendDescription from this location (https://github.com/search?utf8=%E2%9C%93&q=%22flocker.node.script%22+extension%3Apy&type=Code&ref=searchresults) So it should be safe to move.